plugins/test/unit/plugins/distributors/test_configuration.py

```diff
@@ -98,14 +98,14 @@ def test_repo_regisrty_id_with_multiple_slashes(self):
 
     def test_invalid_repo_registry_id(self):
         config = PluginCallConfiguration({
-            constants.CONFIG_KEY_REPO_REGISTRY_ID: 'noUpperCase'
+            constants.CONFIG_KEY_REPO_REGISTRY_ID: 'NOUPPERCASE'
         }, {})
         repo = Mock(id='repoid')
         assert_validation_exception(configuration.validate_config,
                                     [error_codes.DKR1005], config, repo)
 
         config2 = PluginCallConfiguration({
-            constants.CONFIG_KEY_REPO_REGISTRY_ID: 'Nouppsercase'
+            constants.CONFIG_KEY_REPO_REGISTRY_ID: 'NOUPPERCASE'
```

Comment:
It seems like you're testing the same all-uppercase id both on line 101 and 108.